pkg/types/baremetal/platform.go

```diff
@@ -9,6 +9,11 @@ type Platform struct {
 	// Default is qemu:///system
 	URI string `json:"URI,omitempty"`
 
+	// MasterVIP is the VIP for the master API.
+	// +optional
+	// FIXME: bare metal - add a default, use net.IP type
+	MasterVIP string `json:"masterVIP,omitempty"`
```

Comment:
This probably should go to a dedicated `network` config

Reply:
Do you mean the networkConfig? I think I agree with Mark here on having it as platform install config.